x/ecocredit/server/msg_server.go

```diff
@@ -13,6 +13,10 @@ import (
 	"github.com/regen-network/regen-ledger/x/ecocredit"
 )
 
+// TODO: Revisit this once we have proper gas fee framework.
+// Tracking issues https://github.com/cosmos/cosmos-sdk/issues/9054, https://github.com/cosmos/cosmos-sdk/discussions/9072
+const gasCostPerIteration = uint64(20)
```

Comment:
just out of curiosity, what is the rationale for the value here?

Reply:
oh nvm, just saw it in the issue. i think it could be a tad lower, at face value it doesn't look like the msg_server methods here are doing as much as the functions where group gas cost is invoked, but maybe we don't even need this much granularity

Reply:
I've changed it to be `uint64(10)` as in x/authz and x/feegrant, let me know whether we should still adjust or not.